### PR TITLE
add debayer, interpolation methods args

### PR DESCRIPTION
**Pull request by k-okada**
worked on old TODO list

```
-  <!-- TODO Arguments for debayer, interpolation methods? -->
+  <arg name="approximate_sync" default="false" />
+  <arg name="debayer_method" default="0" doc="Debayering algorithm:
+    0 : Bilinear, Fast algorithm using bilinear interpolation
+    1 : EdgeAware, Edge-aware algorithm
+    2 : EdgeAwareWeighted, Weighted edge-aware algorithm
+    3 : VNG, Slow but high quality Variable Number of Gradients algorithm" />
```

### CODE REVIEW

File: image_proc/launch/image_proc.launch [hunk 1]
```diff
@@ -3,7 +3,12 @@
 
   <arg name="manager" /> <!-- Must be globally qualified -->
   <arg name="respawn" default="false" />
-  <!-- TODO Arguments for debayer, interpolation methods? -->
+  <arg name="approximate_sync" default="false" />
+  <arg name="debayer_method" default="0" doc="Debayering algorithm:
```
Comment (SteveMacenski) [inline image_proc/launch/image_proc.launch:7]: Hi,
I don't see where this is used in the pipeline (or even in this launch file). Can you clarify what you're trying to expose?

**Comment by JWhitleyWork** [comment]
@k-okada - Please rebase on `melodic`.

**Comment by k-okada** [comment]
@JWhitleyAStuff it seems you already rebased on melodic. If there is anything i can do, please let me know.

**Comment by JWhitleyWork** [comment]
@k-okada - Please see my comment here: https://github.com/ros-perception/image_pipeline/pull/341#issuecomment-486927270

**Comment by JWhitleyWork** [comment]
Closing due to lack of response.

**Pull request closed without merging**